src/main/kotlin/me/partlysanestudios/partlysaneskies/PartlySaneSkies.kt

```diff
@@ -225,8 +224,10 @@ class PartlySaneSkies {
         registerEvent(SanityCheck)
         registerEvent(Keybinds)
         registerEvent(HealerAlert)
-        registerEvent(WaypointManager)
-        registerEvent(WaypointRender)
+        registerEvent(WaypointEvents)
```

Comment:
I feel like we should sort this big loading thingy at some point, not really needed in here